bazaarvoice/emodb
# EMO-7023 extract blob metadata provider interface

EMO-7023

## What Are We Doing Here?

extract blob metadata provider interface

## How to Test and Verify

1. Check out this PR
2. `mvn clean verify`
3. run `emodb-gatekeeper` tests against this branch

## Risk

### Level 

`Low`, existing flow hasn't been changed, just refactoring
### Required Testing

`Manual`

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
